docs/extend/extend-zowe-overview.md

```diff
@@ -1,56 +1,60 @@
 
-## Adding a plug-in to the Zowe Desktop
+## Add a plug-in to the Zowe Desktop
 
-The Zowe Desktop allows a user to interact with z/OS applications through a web browser.  It is served by the Zowe Application Framework Server on z/OS, also known as Z Lightweight User Experience (ZLUX).  The Zowe desktop comes with a set of default applications. You can extend it to add new applications. For more information, see [Developing for Zowe Application Framework](extend-desktop/mvd-extendingzlux.md).  
+The Zowe Desktop allows a user to interact with z/OS applications through a web browser. The Desktop is served by the Zowe Application Framework Server on z/OS, also known as Z Lightweight User Experience (ZLUX). The Zowe desktop comes with a set of default applications. You can extend it to add new applications. For more information, see [Developing for Zowe Application Framework](extend-desktop/mvd-extendingzlux.md).
 
-The Zowe Desktop is an angular application that allows native plug-ins to be built that enjoy a high level of interoperability with other desktop components.  The React JavaScript toolkit is also supported.  In addition, an existing web application can be included in the Zowe Desktop using an iframe.  
+The Zowe Desktop is an angular application that allows native plug-ins to be built that provide for a high level of interoperability with other desktop components.  The React JavaScript toolkit is also supported. Additionally, you can include an existing web application in the Zowe Desktop using an iframe.
 
-- iframe
-   
-   See [Sample iframe App](extend-desktop/mvd-extendingzlux.md#sample-iframe-app).
-- Angular App
-   
-   See [Sample Angular App](extend-desktop/mvd-extendingzlux.md#sample-angular-app).
-- React App
+**Notes:** For more information, see the following samples:
 
-   See [Sample React App](extend-desktop/mvd-extendingzlux.md#sample-react-app).
+- [Sample iframe App](extend-desktop/mvd-extendingzlux.md#sample-iframe-app).
+- [Sample Angular App](extend-desktop/mvd-extendingzlux.md#sample-angular-app).
+- [Sample React App](extend-desktop/mvd-extendingzlux.md#sample-react-app).
 
-## Lifecycling extensions as Zowe address spaces
+## Lifecycle extensions as Zowe address spaces
 
-Zowe is run under the started task `ZWESVSTC` that brings up its address spaces.  It is possible to introduce a new micro service to be started and stopped with the Zowe stated task. For more information, see [Lifecycling with Zowe](lifecycling-with-zwesvstc.md).
```

Comment:
We discussed that the Zowe lifecycle task information probably doesn't belong to the extensions overview.